tests/python/unittest/test_symbol.py

```diff
@@ -414,6 +416,47 @@ def test_gen_atomic_symbol_multiple_outputs():
                    bidirectional=True, state_outputs=True, mode='lstm')
     atomic_sym = s._gen_atomic_symbol()
 
+
+def test_load_save_symbol():
+    batch_size = 10
+    num_hdidden = 128
+    num_features = 784
+
+    def get_net():
+        data = mx.sym.var('data')
+        weight = mx.sym.var('weight', shape=(num_hdidden, 0))
+        return mx.sym.FullyConnected(data, weight, num_hidden=num_hdidden)
+
+    for flag1 in [False, True]:
```

Comment:
Maybe we need better names for `flag1` and `flag2`?

Reply:
I will rename them in another PR to save the CI cycle for this one.

Reply:
I will rename them in another PR to save the CI cycle for this one.

Reply:
Okay, I'll resolve this one for now.